tests/foreman/ui/test_reporttemplates.py

```diff
@@ -432,11 +435,14 @@ def test_positive_schedule_generation_and_get_mail(
         f'send "q\\r"\n'
     )
 
-    default_sat.execute(f"expect -c '{expect_script}'")
+    result = default_sat.execute(f"expect -c '{expect_script}'")
+    assert result.status == 0
```

Comment:
Combine this in 1 as result var isnt used again.
```suggestion
    assert default_sat.execute(f"expect -c '{expect_script}'").status == 0
```